front/commonitilvalidation.form.php

```diff
@@ -80,6 +80,39 @@
                 sprintf(__('%s adds an approval'), $_SESSION["glpiname"])
             );
         }
+    } else if ($_POST['validatortype'] == 'requester_responsible') {
+        if (!isset($_POST['itemtype'])) {
+            Html::back();
+        }
+        $itemtype = $_POST['itemtype'];
+        $itemtypeFk = $itemtype::getForeignKeyField();
+        if (!isset($_POST[$itemtypeFk])) {
+            Html::back();
+        }
+
+        $itilObject = new $itemtype();
+        if (!$itilObject->getFromDB($_POST[$itemtypeFk])) {
+            Html::back();
+        }
+        $primaryRequester = $itilObject->getPrimaryRequesterUser();
+        if ($primaryRequester === null) {
+            Html::back();
+        }
+        if ($primaryRequester->fields['users_id_supervisor'] == 0) {
+            // TRANS: $1%s us the friendly user name
+            Session::addMessageAfterRedirect(sprintf(__('%1$s does not have a responsible'), $primaryRequester->getFriendlyName()));
```

Comment:
```suggestion
            Session::addMessageAfterRedirect(sprintf(__('%1$s does not have a supervisor'), $primaryRequester->getFriendlyName()));
```
Sorry if I wasn't clear, but "Supervisor" is the word that made the most sense to me. In the user form currently, it is labelled "Responsible", but that is an adjective and doesn't make any sense as a label in English.

Reply:
see #12707